Adyen/UI/View Controllers/AddressLookup/AddressLookupViewController.swift

```diff
@@ -112,4 +95,19 @@ private extension AddressLookupViewController.ViewModel {
             completionHandler: handleAddressInputFormCompletion(validAddress:)
         )
     }
+    
+    func addressSearchViewModel(
```

Comment:
Minor: function name supposed to start with verb  🤓

How about:
`getAddressSearchViewModel` or `buildAddressSearchViewModel` ?

Same applicable to `func addressInputFormViewModel`.

Reply:
Yes! Sounds good, will change 👌

Reply:
Fixed in c38ed109abaa17814809d01660c6cbf26f83ca47